src/ol_infrastructure/applications/bootcamp/__main__.py

```diff
@@ -0,0 +1,113 @@
+import pulumi
+import pulumi_aws as aws
+
+bucket = aws.s3.Bucket(
+    "bucket",
+    arn="arn:aws:s3:::bootcamp-static-assets-test",
+    bucket="bootcamp-static-assets-test",
+    hosted_zone_id="Z3AQBSTGFYJSTF",
+    request_payer="BucketOwner",
+    tags={
+        "OU": "bootcamps",
+        "business_unit": "bootcamps",
+    },
+    opts=pulumi.ResourceOptions(protect=True),
+)
+
+bucket_app_ci = aws.s3.Bucket(
+    "bucket_app_ci",
+    arn="arn:aws:s3:::ol-bootcamps-app-ci",
+    bucket="ol-bootcamps-app-ci",
+    hosted_zone_id="Z3AQBSTGFYJSTF",
+    request_payer="BucketOwner",
+    tags={
+        "Department": "bootcamps",
+        "Environment": "ci",
+        "OU": "bootcamps",
+        "business_unit": "bootcamps",
+    },
+    versioning=aws.s3.BucketVersioningArgs(
+        enabled=True,
+    ),
+    opts=pulumi.ResourceOptions(protect=True),
+)
+
+ol_bootcamps_app_production = aws.s3.Bucket(
+    "ol_bootcamps_app_production",
+    arn="arn:aws:s3:::ol-bootcamps-app-production",
+    bucket="ol-bootcamps-app-production",
+    hosted_zone_id="Z3AQBSTGFYJSTF",
+    request_payer="BucketOwner",
+    tags={
+        "Department": "bootcamps",
+        "Environment": "production",
+        "OU": "bootcamps",
+        "business_unit": "bootcamps",
+    },
+    versioning=aws.s3.BucketVersioningArgs(
+        enabled=True,
+    ),
+    opts=pulumi.ResourceOptions(protect=True),
+)
+
+ol_bootcamps_app_rc = aws.s3.Bucket(
+    "ol_bootcamps_app_rc",
+    arn="arn:aws:s3:::ol-bootcamps-app-rc",
+    bucket="ol-bootcamps-app-rc",
+    hosted_zone_id="Z3AQBSTGFYJSTF",
+    request_payer="BucketOwner",
+    tags={
+        "Department": "bootcamps",
+        "Environment": "rc",
```

Comment:
Why are the CI, RC and production buckets being defined (and hardcoded) in the same stack?

Reply:
I thought that since we are importing these buckets we can keep  them in the same stack.

Reply:
The names should not be hardcoded? How else can we put names for these? I put them in the same stack since they were imported. Can you define the definition of done on these? That way I can get this done neatly instead of doing it and then you commenting on it. I thought all you wanted was an import.

Reply:
Can we add like a prefix based on the env?